playground/src/TabPane.tsx

```diff
@@ -96,4 +98,18 @@ export class TabPane extends React.Component<ITabPaneProps, ITabPaneState>  {
     this.setState({ selectedTabIndex: tabIndex });
   }
 
+  private getTabIndexFromUrl(): number {
+    const { tabs } = this.props;
+
+    let index: number = this.DEFAULT_TAB_INDEX;
+
+    tabs.forEach((tab: ITabDefinition, idx: number, _tabs: ITabDefinition[]) => {
+      const urlHashTitle: string = document.location.hash.substring(1, document.location.hash.length);
+      if (tab.title.toLowerCase() === urlHashTitle.toLowerCase()) {
```

Comment:
>toLowerCase [](start = 51, length = 11)

For case insensitive string comparisons, use `toUpperCase()` because (1) it handles Turkish "I" correctly and (2) when saved in a variable it makes it more obvious that the case has been normalized